pages/shared/data/tools.yaml

```diff
@@ -466,3 +466,14 @@ providers:
       fit: contain
     formats:
       - email
+  - name: WompMobile
+    url: https://www.wompmobile.com
+    type: Development
+    description: WompMobile is an Accelerator Platform to develop, validate, launch and manage enterprise-grade AMP experiences.
```

Comment:
Can you rephrase in a way to focus on the tooling aspect?

Reply:
Updated description to specifically mention use of tools: "Built-in tools monitor for errors, ensure validation, purge the AMP cache, and create Signed-Exchange and Server-Side-Rendered AMPs."